fetch.bs

```diff
@@ -4903,8 +4951,60 @@ initially a new {{AbortSignal}} object.
  <var>init</var>])</code>
  <dd>Returns a new <var>request</var> whose {{Request/url}} property is <var>input</var> if
  <var>input</var> is a string, and <var>input</var>'s {{Request/url}} if <var>input</var> is a
- {{Request}} object. The optional <var>init</var> argument allows for setting attributes
- appearing in {{RequestInit}} via object members of the same name.
+ {{Request}} object.
+
+ <p>The optional <var>init</var> argument allows for setting properties appearing in {{RequestInit}}
+ via object members of the same name. These are the object members that can be used:</p>
+
+ <dl>
+  <dt>{{RequestInit/method}}
+  <dd>Set to a string to set <var>request</var>'s {{Request/method}}.
+
+  <dt>{{RequestInit/headers}}
+  <dd>Set to a {{Headers}} object or an array of two-item arrays to set <var>request</var>'s
+  {{Request/headers}}.
+
+  <dt>{{RequestInit/body}}
+  <dd>Set to a {{Body}} object or null to set <var>request</var>'s <a>body</a>.
+
+  <dt>{{RequestInit/referrer}}
+  <dd>Set to a string whose value is a same-origin URL, "<code>about:client</code>", or the empty
+  string, to set <var>request</var>'s <a>referrer</a>.
+
+  <dt>{{RequestInit/referrerPolicy}}
+  <dd>Set to a <a for=/>referrer policy</a> to set <var>request</var>'s {{Request/referrerPolicy}}.
+
+  <dt>{{RequestInit/mode}}
+  <dd>Set to a string to indicate whether the request will use CORS, or will be restricted to
+  same-origin URLs. Sets <var>request</var>'s {{Request/mode}}.
+
+  <dt>{{RequestInit/credentials}}
+  <dd>Set to a string indicating whether credentials will be sent with the request always, never, or
+  only when sent to a same-origin URL. Sets <var>request</var>'s {{Request/credentials}}.
+
+  <dt>{{RequestInit/cache}}
+  <dd>Set to a string indicating how the request will interact with the browser's cache to set
+  <var>request</var>'s {{Request/cache}}.
+
+  <dt>{{RequestInit/redirect}}
+  <dd>Set to a string indicating whether or not <var>request</var> should automatically follow
+  redirects, result in an error, or manually follow redirects. Sets <var>request</var>'s
+  {{Request/redirect}}.
+
+  <dt>{{RequestInit/integrity}}
+  <dd>Set to a cryptographic has of the resource to be fetched by <var>request</var>. Sets
+  <var>request</var>'s {{Request/integrity}}.
+
+  <dt>{{RequestInit/keepalive}}
+  <dd>Set to a boolean to set <var>request</var>'s {{Request/keepalive}}.
+
+  <dt>{{RequestInit/signal}}
+  <dd>Set to an {{AbortSignal}} to set <var>request</var>'s {{Request/signal}}.
+
+  <dt>{{RequestInit/window}}
+  <dd>Can only be null. Used to set <var>request</var>'s internal <a>window</a> object to
```

Comment:
Not an object. I'd maybe say "Used to de-associate the request from any {{Window}}"?